### PR TITLE
Update time-range-controls.md

### PR DESCRIPTION
**Pull request by castillo92**
Adding more examples of relative time instructions, it could hep to newbie people to understand it.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by achatterjee-grafana — APPROVED** [review]
Looks Ok to me, will need Dev approval.

File: docs/sources/dashboards/time-range-controls.md [hunk 1]
```diff
@@ -26,8 +26,12 @@ Here are some examples:
 | Last 5 minutes         | `now-5m`   | `now`      |
 | The day so far         | `now/d`    | `now`      |
 | This week              | `now/w`    | `now/w`    |
-| Week to date           | `now/w`    | `now`      |
+| This week so far       | `now/w`    | `now`      |
+| This month             | `now/M`    | `now/M`    |
+| This month so far      | `now/M`    | `now`      |
 | Previous Month         | `now-1M/M` | `now-1M/M` |
```
Comment (oddlittlebird) [inline docs/sources/dashboards/time-range-controls.md:32]:
```suggestion
| Previous month         | `now-1M/M` | `now-1M/M` |
```

File: docs/sources/dashboards/time-range-controls.md [hunk 2]
```diff
@@ -26,8 +26,12 @@ Here are some examples:
...
+| This year so far       | `now/Y`    | `now`      |
+| This Year              | `now/Y`    | `now/Y     |
```
Comment (oddlittlebird) [inline docs/sources/dashboards/time-range-controls.md:34]:
```suggestion
| This year              | `now/Y`    | `now/Y     |
```

**Review by oddlittlebird — COMMENTED** [review]
A couple minor edits

**Comment by achatterjee-grafana** [comment]
Good catch @oddlittlebird, thanks!

**Review by dprokop — APPROVED** [review]

**Pull request merged**